hawkular-alerts-engine/src/main/java/org/hawkular/alerts/engine/impl/PartitionManagerImpl.java

```diff
@@ -79,6 +81,7 @@
  *          <local-cache name="partition"/>
  *          <local-cache name="triggers"/>
  *          <local-cache name="data"/>
+ *          <local-cache name="definitionsEvents"/>
```

Comment:
sorry, but maybe lose the double plural and make this "definitionEvents".   Adding caches causes a lot of PRs to update things in various places, but I guess it's unavoidable.